packages/paypal-commerce-utils/src/utils/is-paypal-commerce-connect-window.ts

```diff
@@ -0,0 +1,7 @@
+import { PayPalCommerceHostWindow } from '../paypal-commerce-types';
+
+export default function isPayPalCommerceConnectWindow(
+    window: Window,
+): window is PayPalCommerceHostWindow {
+    return Boolean(window.hasOwnProperty('paypalConnect'));
```

Comment:
`hasOwnProperty` already returns a boolean value. i think you can remove `Boolean` in this case